app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,13 @@
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
 
+    <% # Select2 (select2.org) main distro, locale files, CSS %>
+    <%= javascript_include_tag "https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/js/select2.min.js" %>
```

Comment:
Are we OK with using remote-hosted scripts? 
Would cdn.jsdelivr.net take OTIS down if it crashed (and user did not have it cached) ?

Reply:
LIT is using `unpkg.com` as a CDN, although we're exploring ways to cascade from `unpkg.com` failures (which did happen recently).

Reply:
We depend on remote Bootstrap and Bootstrap-table, so this one going poof is no worse than those two. App would totally look like crap but it would still function. Ckeditor (the inline e-mail editor) is also remote, and that one would actually cripple OTIS if it disappeared. I fear this is just the Achilles heel of the state of the art in web apps.